From: KevinDuan <duankaiwen@kylinos.cn>
Date: Wed, 26 Aug 2026 07:35:33 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (42 of 42 strings)

Translation: openkylin-huanghe/peony-share
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-share/ko_KP/
---
 peony-share/app/translations/peony-share-extension_ko_KP.ts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/peony-share/app/translations/peony-share-extension_ko_KP.ts b/peony-share/app/translations/peony-share-extension_ko_KP.ts
index 07ed698..ef64f99 100644
--- a/peony-share/app/translations/peony-share-extension_ko_KP.ts
+++ b/peony-share/app/translations/peony-share-extension_ko_KP.ts
@@ -32,7 +32,7 @@
     <message>
         <location filename="../advanced-share-page.cpp" line="127"/>
         <source>Tips</source>
-        <translation>알림</translation>
+        <translation>팁</translation>
     </message>
     <message>
         <location filename="../advanced-share-page.cpp" line="127"/>
@@ -47,7 +47,7 @@
     <message>
         <location filename="../advanced-share-page.cpp" line="274"/>
         <source>User</source>
-        <translation>사용자 이름</translation>
+        <translation>사용자명</translation>
     </message>
     <message>
         <location filename="../advanced-share-page.cpp" line="274"/>
@@ -72,7 +72,7 @@
     <message>
         <location filename="../advanced-share-page.cpp" line="383"/>
         <source>delete</source>
-        <translation>백업 삭제</translation>
+        <translation>제거</translation>
     </message>
     <message>
         <location filename="../advanced-share-page.cpp" line="400"/>
@@ -82,7 +82,7 @@
     <message>
         <location filename="../advanced-share-page.cpp" line="417"/>
         <source>Save</source>
-        <translation>로 스캔</translation>
+        <translation>확정</translation>
     </message>
     <message>
         <location filename="../advanced-share-page.cpp" line="418"/>
@@ -240,7 +240,7 @@
     <message>
         <location filename="../share-page.cpp" line="606"/>
         <source>Allow others to create and delete files</source>
-        <translation type="unfinished"></translation>
+        <translation>다른 사람이 파일을 작성하고 삭제하는것을 허용합니다</translation>
     </message>
     <message>
         <source>Read Only</source>
